lib/DxilPIXPasses/PixPassHelpers.cpp

```diff
@@ -500,6 +500,90 @@ unsigned int FindOrAddSV_Position(hlsl::DxilModule &DM,
   }
 }
 
+void ForEachDynamicallyIndexedResource(
+    hlsl::DxilModule &DM,
+    const std::function<bool(bool, Instruction *, Value *)> &Visitor) {
+  OP *HlslOP = DM.GetOP();
+  LLVMContext &Ctx = DM.GetModule()->getContext();
+
+  for (llvm::Function &F : DM.GetModule()->functions()) {
+    if (F.isDeclaration() && !F.use_empty() && OP::IsDxilOpFunc(&F)) {
+      if (F.hasName()) {
+        if (F.getName().find("createHandleForLib") != StringRef::npos) {
```

Comment:
There's probably a good reason that I've forgotten , but why is createHandleForLib treated differently from the other three in this function?

Reply:
Oh, cuz it got pasted from the access tracking pass! Might be a good opportunity to rationalize the four cases, then.

Reply:
(Decided that would be for another day)